release/build_release.sh

```diff
@@ -58,125 +58,170 @@ function check_usage {
 
-		lc_time_run obfuscate_licensing
+	lc_time_run set_up_profile
 
-		lc_time_run build_product
+	lc_time_run add_licensing
 
-		lc_background_run build_sql
-		lc_background_run copy_copyright
-		lc_background_run deploy_elasticsearch_sidecar
-		lc_background_run clean_up_ignored_dxp_modules
-		lc_background_run clean_up_ignored_dxp_plugins
+	lc_time_run compile_product
 
-		lc_wait
+	lc_time_run obfuscate_licensing
 
-		lc_time_run warm_up_tomcat
+	lc_time_run build_product
 
-		lc_time_run install_patching_tool
+	lc_background_run build_sql
+	lc_background_run copy_copyright
+	lc_background_run deploy_elasticsearch_sidecar
+	lc_background_run clean_up_ignored_dxp_modules
+	lc_background_run clean_up_ignored_dxp_plugins
 
-		lc_time_run generate_api_jars
+	lc_wait
 
-		lc_time_run generate_api_source_jar
+	lc_time_run warm_up_tomcat
 
-		lc_time_run generate_poms
+	lc_time_run install_patching_tool
 
-		generate_poms_from_scratch
+	lc_time_run generate_api_jars
 
-		lc_time_run package_release
+	lc_time_run generate_api_source_jar
 
-		lc_time_run package_boms
+	lc_time_run generate_poms
 
-		lc_time_run generate_checksum_files
+	generate_poms_from_scratch
 
-		lc_time_run generate_release_properties_file
+	lc_time_run package_release
 
-		lc_time_run generate_release_notes
+	lc_time_run package_boms
 
-		lc_time_run upload_boms xanadu
+	lc_time_run generate_checksum_files
 
-		lc_time_run upload_release
-	else
-		lc_time_run prepare_release_dir
+	lc_time_run generate_release_properties_file
 
-		lc_time_run copy_release_info_date
+	lc_time_run generate_release_notes
 
-		lc_time_run set_up_profile
+	lc_time_run upload_boms xanadu
 
-		lc_time_run add_hotfix_testing_code
+	lc_time_run upload_release
+}
 
-		lc_time_run set_hotfix_name
+function main {
+	export ANT_OPTS="-Xmx10G"
+
+	print_variables
+
+	check_usage
+
+	lc_time_run configure_jdk
 
-		lc_time_run add_licensing
+	lc_time_run report_jenkins_url
 
-		lc_time_run compile_product
+	lc_background_run clone_repository liferay-binaries-cache-2020
+	lc_background_run clone_repository liferay-portal-ee
+	lc_background_run clone_repository liferay-release-tool-ee
 
-		lc_time_run obfuscate_licensing
+	lc_wait
 
-		lc_time_run build_product
+	lc_time_run clean_portal_repository
 
-		lc_time_run clean_up_ignored_dxp_modules
+	lc_background_run init_gcs
+	lc_background_run update_portal_repository
 
-		lc_time_run clean_up_ignored_dxp_plugins
+	lc_wait
 
-		lc_time_run add_portal_patcher_properties_jar
+	lc_time_run set_git_sha
 
-		lc_time_run create_hotfix
+	lc_background_run decrement_module_versions
+	lc_background_run update_release_tool_repository
 
-		lc_time_run calculate_checksums
+	lc_wait
 
-		lc_time_run create_documentation
+	lc_time_run set_product_version
 
-		lc_time_run sign_hotfix
+	case "${LIFERAY_RELEASE_OUTPUT}" in
+		"hotfix")
+			execute_hotfix
 
-		lc_time_run package_hotfix
+			;;
+		"nightly")
+			execute_nightly
 
-		lc_time_run upload_hotfix
+			;;
+		*)
```

Comment:
@Wykenz, can we have a dedicated entry to `release`?

The idea is prevent undesired code executions for future code changes :slightly_smiling_face: